app/models/organization.rb

```diff
@@ -39,4 +41,10 @@ def generate_token
       break random_token unless Organization.exists?(token: random_token)
     end
   end
+
+  # def send_notification_about_creation
```

Comment:
fyi: `notify_user_created!` or perhaps `notify_created!`

also, move this to a concern as it's all the same logic except notification type, which is pretty easy to detect by `self.class` for example